framework/src/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -105,6 +105,16 @@
     private final Map<Element, Boolean> elementAnnotatedFors = new IdentityHashMap<>();
 
     /**
+     * CLIMB-to-the-top locations whose standard default is top
+     */
+    public static final DefaultLocation[] standardClimbDefaultsTop = { DefaultLocation.LOCAL_VARIABLE, DefaultLocation.RESOURCE_VARIABLE,
```

Comment:
Is this a comment about the name of this field or a more general comment about CLIMB-to-top?  I used CLIMB in the code because that is how the standard checked code defaults are described in the manual.